modules/rest-api/src/main/java/org/wso2/carbon/mdm/api/Operation.java

```diff
@@ -93,8 +91,9 @@ public Message addOperation(DeviceOperationContext operationContext) throws MDMA
 	}
 
     @POST
-    @Path("installApp")
-    public Message installApplication(ApplicationWrapper applicationWrapper) throws MDMAPIException {
+    @Path("installApp/{tenantDomain}")
```

Comment:
This is added to fulfil a requirement which is requested by AppM. The reason to add tenantDomain in here is to install specific set of apps to an entire tenant as a bulk operation.